Makefile

```diff
@@ -141,14 +141,10 @@ package: build-if-changed
 
 .PHONY: package-win32
 package-win32:
-	@echo Packaging .exe...
+	@echo "Packaging exe..."
 	@npx electron-builder --win -p $(PUBLISH)
-ifeq ($(IS_WINDOWS),true)
-	@echo Packaging .appx as well...
+	@echo "Packaging appx with dedicated configuration to work around code signing conflicts..."
```

Comment:
A tiny attempt of help future readers understand the need for the dedicated config